src/yunohost/app.py

```diff
@@ -829,12 +829,110 @@ def app_makedefault(auth, app, domain=None):
         raise MoulinetteError(errno.EPERM,
                               m18n.n('ssowat_persistent_conf_write_error', error=e.strerror))
 
+    os.system('chmod 644 /etc/ssowat/conf.json.persistent')
```

Comment:
Uses https://docs.python.org/2/library/os.html#os.chmod instead and check for file existance before to avoid a bug.